app/workers/functions.php

```diff
@@ -270,18 +270,18 @@ private function execute(
 
         /** Collect environment variables */
         $vars = \array_merge($vars, [
-            'APPWRITE_FUNCTION_ID' => $functionId,
+            'APPWRITE_FUNCTION_ID' => $functionId ?? '',
             'APPWRITE_FUNCTION_NAME' => $function->getAttribute('name', ''),
-            'APPWRITE_FUNCTION_DEPLOYMENT' => $deploymentId,
-            'APPWRITE_FUNCTION_RUNTIME_NAME' => $runtime['name'],
-            'APPWRITE_FUNCTION_RUNTIME_VERSION' => $runtime['version'],
-            'APPWRITE_FUNCTION_TRIGGER' => $trigger,
-            'APPWRITE_FUNCTION_EVENT' => $event,
-            'APPWRITE_FUNCTION_EVENT_DATA' => $eventData,
-            'APPWRITE_FUNCTION_DATA' => $data,
-            'APPWRITE_FUNCTION_PROJECT_ID' => $project->getId(),
-            'APPWRITE_FUNCTION_USER_ID' => $user->getId(),
-            'APPWRITE_FUNCTION_JWT' => $jwt,
+            'APPWRITE_FUNCTION_DEPLOYMENT' => $deploymentId ?? '',
```

Comment:
Variables like functionID and projectID are never empty, so we can leave them as it is